astroid/brain/brain_multiprocessing.py

```diff
@@ -78,6 +78,7 @@ class SyncManager(object):
         Queue = JoinableQueue = queue.Queue
         Event = threading.Event
         RLock = threading.RLock
+        Lock = threading.Lock
```

Comment:
```suggestion
        Lock = multiprocessing.synchronize.Lock()
```
Would this work ? I'm basing this suggestion on https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Lock

Reply:
Yay this indeed works perfectly! Thank you!